utils/conversions.ts

```diff
@@ -116,9 +117,17 @@ export function retrievedServiceToVisibleService(
     serviceId,
     serviceMetadata,
     serviceName,
-    version
+    version,
+    _etag,
+    _rid,
+    _self,
+    _ts
   } = retrievedService;
   return {
+    _etag,
+    _rid,
+    _self,
+    _ts,
```

Comment:
```suggestion
```

Reply:
same as above :)

Reply:
ditto

Reply:
fixed in [#e6e2575](https://github.com/pagopa/io-functions-admin/pull/65/commits/e6e2575ed93102aa692f1732c0e859b710b57a7d)